ports/tensorpipe/portfile.cmake

```
@@ -0,0 +1,35 @@
+vcpkg_fail_port_install(ON_TARGET "windows" "uwp")
+vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO pytorch/tensorpipe
+    REF a9aa71a2fe49a8e8475cd5ff16cbd0de13b67c2b
+    SHA512 8b2679d4325acc6cd669326504bb49a645c126586b1d61d873d5a0423ed27e7c513574ccd2e74bdd792869167d3daba0804f127e21cfd5831d37a6d1ef7a79ba
+    PATCHES
+        fix-cmakelists.patch
+)
+
+vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
+  FEATURES
+    cuda        TP_USE_CUDA
+    cuda        TP_ENABLE_CUDA_IPC
+    shm         TP_ENABLE_SHM
+    ibv         TP_ENABLE_IBV
+    cma         TP_ENABLE_CMA
+    pybind11    TP_BUILD_PYTHON
+)
```

```suggestion
vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
    FEATURES
        cuda        TP_USE_CUDA
        cuda        TP_ENABLE_CUDA_IPC
        shm         TP_ENABLE_SHM
        ibv         TP_ENABLE_IBV
        cma         TP_ENABLE_CMA
        pybind11    TP_BUILD_PYTHON
)
```
There should be 4 spaces.